anonymize.py

```diff
@@ -232,10 +230,11 @@ def unzip_file(orig_file, tmp_dir):
 
 def cycle_ask(cur_files):
     '''opens and reads file to be changed, asks for input until user is fine
-    then writes the changed string, until you call it quits. This is sort of the
-    main function here'''
+    then writes the changed string, until you call it quits. This is sort of
+    the main function here'''
 
     # Flatten list of lists structure
+    # FIXME filelist variable declared but not used. Missing anything?
     filelist = [item for subfile in cur_files for item in subfile.textfiles]
```

Comment:
You're right, this can be removed. I think I forgot to take it out when I was testing a different approach